etc/samples/tendrl_definitions_generic-0.1.sample.yaml

```diff
@@ -0,0 +1,77 @@
+---
+valid_objects:
+- volume
+
+object_details:
+  volume:
+    enabled: True
+    value: "/clusters/gluster/$cluster.id/volumes/$volume.id
+    attrs:
+      volname:
+        type: String
+        help: "Name of the gluster volume"
+      stripe_count:
+        type: Integer
+        help: "Volume Stripe count"
+      replica_count:
+        type: Integer
+      disperse_count:
+        type: Integer
+      redundancy_count:
+        type: Integer
+      transport:
+        type: String
+      brickdetails:
+        type: Brick[]
+      force:
+        type: Boolean
+      directory:
+        type: String
+      bitrot:
+        type: Boolean
+      scrub-frequency:
+        type: Integer
+      scrub-throttle:
+        type: Integer
+    atoms:
+        create:
+          enabled: True
+          uuid: statically generated per atom
+          # List of flows this of which this atom can be part of
+          flows: ["CreateGlusterVolume"]
```

Comment:
tendrl-api exposes objects and the atoms valid for those objects, there needs to be a way we map these atoms to flows (eg: create gluster volume atom can be run via createglustervolume flow or the createglustervolumewithoutbricks flow) based on tendrl-api job inputs, hence required for now.